quasar/main.tf

```diff
@@ -615,3 +615,9 @@ module "storage" {
   acl        = "private"
   versioning = true
 }
+
+module "fivetran_logging" {
+  source = "../components/fivetran_cloudwatch_role"
+
```

Comment:
This module requires a `fivetran_cloudwatch_integration_external_id ` argument (see `variables.tf` above):

```hcl
module "fivetran_logging" {
  source = "../components/fivetran_cloudwatch_role"
  fivetran_cloudwatch_integration_external_id = "..."
}
```